app.js

```diff
@@ -136,17 +135,16 @@ if (Config.crashguard) {
 
 global.Sockets = require('./sockets');
 
-exports.listen = function (port, bindAddress, workerCount) {
-	Sockets.listen(port, bindAddress, workerCount);
+exports.listen = function (port) {
```

Comment:
Just for the off chance someone tries to use an IPv6 IP as the bind address when it's not supported (yet)